slug.go

```diff
@@ -47,18 +47,34 @@ func ApplyTerraformIgnore() PackerOption {
 }
 
 // DereferenceSymlinks is a PackerOption that will allow symlinks that
-// reference a target outside of the src directory.
+// reference a target outside of the source directory by copying the link
+// target, turning it into a normal file within the archive.
 func DereferenceSymlinks() PackerOption {
 	return func(p *Packer) error {
 		p.dereference = true
 		return nil
 	}
 }
 
+// AllowSymlinkTarget relaxes safety checks on symlinks with targets matching
+// path. Specifically, absolute symlink targets (e.g. "/foo/bar") and relative
+// targets (e.g. "../foo/bar") which resolve to a path outside of the
+// source/destination directories for pack/unpack operations respectively, may
+// be expressly permitted, whereas they are forbidden by default. Exercise
+// caution when using this option. A symlink matches path if its target
+// resolves to path exactly, or if the target is prefixed by path.
+func AllowSymlinkTarget(path string) PackerOption {
+	return func(p *Packer) error {
+		p.allowSymlinkTargets = append(p.allowSymlinkTargets, path)
```

Comment:
Per our chat, let's avoid the ambiguity of partial file path prefixes, by appending a slash if one wasn't provided.

Reply:
This case should be resolved by the latest commit. We can still allow exact matches or specifically directory prefixes.